velox/type/Timestamp.h

```diff
@@ -67,8 +67,14 @@ struct TimestampToStringOptions {
   };
 
   Mode mode = Mode::kFull;
+
+  const date::time_zone* timeZone = nullptr;
```

Comment:
nit: maybe use std::optional instead of a raw pointer

Reply:
It generates below exception if using std::optional\<date::time_zone\>.

> ‘value’ is not a member of ‘std::is_trivially_copy_constructible\<facebook::velox::date::time_zone\>’